### PR TITLE
fix: Render item in wishlist properly if ID is a string

### PR DESCRIPTION
**Pull request by acasazza**

### CODE REVIEW

File: components/wishlist/WishlistButton/WishlistButton.tsx [hunk 1]
```diff
@@ -32,7 +32,9 @@ const WishlistButton: FC<Props> = ({
     // @ts-ignore Wishlist is not always enabled
     (item) =>
       item.product_id === Number(productId) &&
-      (item.variant_id as any) === Number(variant.id)
+      (item.variant_id as any) === Number(variant.id) ||
+      item.product_id === productId &&
+      (item.variant_id as any) === variant.id
```
Comment (goncy) [inline components/wishlist/WishlistButton/WishlistButton.tsx:37]: Just doing the checks without Number casting should be ok

Comment (Randagio13) [reply components/wishlist/WishlistButton/WishlistButton.tsx:37]: Ok perfect! I will edit shortly

Comment (acasazza) [reply components/wishlist/WishlistButton/WishlistButton.tsx]: Done

**Review by goncy — APPROVED** [review]

**Comment by goncy** [comment]
@acasazza @Randagio13 we now need that all commits are signed, can you please do it? https://docs.github.com/en/authentication/managing-commit-signature-verification/about-commit-signature-verification thanks!

**Review by goncy — APPROVED** [review]

**Pull request merged**